shared-libs/cht-script-api/test/index.spec.js

```diff
@@ -35,13 +42,15 @@ describe('CHT Script API - index', () => {
       can_roll_over: [ 'national_admin', 'district_admin' ],
     };
     const permissions = [
-      ['can_backup_facilities'],
-      ['can_export_messages', 'can_roll_over'],
-      ['can_add_people', 'can_add_places'],
+      [ 'can_backup_facilities' ],
+      [ 'can_export_messages', 'can_roll_over' ],
+      [ 'can_add_people', 'can_add_places' ],
     ];
 
     const result = chtScriptApi.v1.hasAnyPermission(permissions, userRoles, chtPermissionsSettings);
 
     expect(result).to.be.true;
+    expect(authHasAnyPermission.callCount).to.equal(1);
+    expect(authHasAnyPermission.args[0]).to.have.members([ permissions, userRoles, chtPermissionsSettings ]);
```

Comment:
Please use `to.deep.equal` here. `have.members` ignores order, which is pretty important in the case of arguments passed.

Reply:
Done!